goodtables/compat.py

```diff
@@ -45,22 +46,28 @@ def iterenc_utf8(data):
 
         iter = chain(first_lines, data)
         iter = iterenc_utf8(iter)
-        reader = csv.reader(iter, dialect=dialect, **kwargs)
-        for row in reader:
-            yield [str(cell, 'utf-8') for cell in row]
+        csv.field_size_limit(20000000)
```

Comment:
It's a _field_ size limit that used to have the default value 131072 which wasn't enough for some of the files so I changed it to a number unlikely to be reached.